lib/SILOptimizer/Transforms/SimplifyCFG.cpp

```diff
@@ -1014,15 +1014,19 @@ bool SimplifyCFG::tryJumpThreading(BranchInst *BI) {
     }
   }
 
+  ThreadingBudget -= JumpThreadingCost[SrcBB];
```

Comment:
Why is `SrcBB` included in the JumpThreadingCost? The transformation duplicated `DestBB`.